File: Sorting Algorithms/insertionSort.py
def insertSort(array):

    for i in range(1, len(array)):
      current=array[i]
      prev=i-1 //8
      while prev>=0 and array[prev] >current:
         array[prev+1]= array[prev]
         prev-=1 //-1
      array[prev+1]=current
    return array



a=[8,6]

insertSort(a)
print(a)

File: Sorting Algorithms/insertionSort.js
function insertSort(arr){
  
    for(i=1; i<arr.length; i++){
      const current=arr[i]
      let previous=i-1

      while (previous>=0 && arr[previous]>current){
        arr[previous+1]=arr[previous]
        previous-=1
      } 

      arr[previous+1]=current

    }
   


}

let array=[9,3,4,33,13,1,70,-10]
insertSort(array)
console.log(array)